### PR TITLE
Add a conda forge recipe for jupyterlab-sos

### PR DESCRIPTION
**Pull request by BoPeng**
Checklist

- [x ] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x ] Source is from official source
- [x ] Package does not vend other packages
- [x ] Build number is 0
- [ x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your
      recipe (see
      [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos)
      for more details)
- [ x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Pending the merge of https://github.com/conda-forge/staged-recipes/pull/9383

### CODE REVIEW

**Comment by BoPeng** [comment]
@conda-forge/help-python This recipe is ready to be reviewed.

File: recipes/jupyterlab-sos/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+
+  url: https://registry.npmjs.org/jupyterlab-sos/-/jupyterlab-sos-{{ version }}.tgz
+  sha256: afbfa0c906a45db6e97b12731d308de89e80990833401355198f75f455328de5
+
+build:
+  number: 0
+  noarch: python
```
Comment (scopatz) [inline recipes/jupyterlab-sos/meta.yaml]: I don't think this is noarch python if there are separate build scripts

Comment (BoPeng) [reply recipes/jupyterlab-sos/meta.yaml]: Ohmm, I agree. Will change soon.

Comment (dbast) [reply recipes/jupyterlab-sos/meta.yaml]: It should be noarch, as the package does not contain any compiled or architecture specific code... The fact that there are separate build scripts is just for the case that somebody wants to build the noarch package on windows... other jupyterlab packages are build the same way and are also noarch.

File: recipes/jupyterlab-sos/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+  url: https://registry.npmjs.org/jupyterlab-sos/-/jupyterlab-sos-{{ version }}.tgz
+  sha256: afbfa0c906a45db6e97b12731d308de89e80990833401355198f75f455328de5
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - nodejs
```
Comment (scopatz) [inline recipes/jupyterlab-sos/meta.yaml]: Should this be in host?

Comment (BoPeng) [reply recipes/jupyterlab-sos/meta.yaml]: I do not know. All other jupyterlab extensions have this in build https://github.com/conda-forge/jupyterlab-git-feedstock/blob/master/recipe/meta.yaml is one of them.

Comment (isuruf) [reply recipes/jupyterlab-sos/meta.yaml]: it should be `build` as it is a build dependency and not a host one.

File: recipes/jupyterlab-sos/meta.yaml (hunk 1 above)
Comment (BoPeng) [reply recipes/jupyterlab-sos/meta.yaml]: Changed back to `noarch`.

File: recipes/jupyterlab-sos/meta.yaml (hunk 2 above)
Comment (BoPeng) [reply recipes/jupyterlab-sos/meta.yaml]: kept in `build`.

**Comment by scopatz** [comment]
seems like there is a conflict here.

**Comment by BoPeng** [comment]
Not sure what went wrong but I have re-pushed the recipe.

**Comment by BoPeng** [comment]
@scopatz Should be ok now.

**Pull request merged**